lib/paths/find-lib.js

```diff
@@ -0,0 +1,16 @@
+import { resolve, dirname, join } from 'path';
```

Comment:
does this diverge much from https://github.com/stefanpenner/broccoli-stew/blob/master/lib/npm.js#L17 ?

If so, thoughts on this living in stew, as it isn't ember related at all. feels more broccoli stdlib'y